README.rst

```diff
@@ -48,6 +48,9 @@ installed using ::
 To install the remaining packages see the installation instructions
 at their respective web pages.
 
+Notice that due to the update from Python2 to Python3, the `.ddp` project
```

Comment:
"Please note that at this time it will not be possible for `.ddp` project files that were saved by the Python2 PDFgui cannot be read by Python3 PDFgui.  We are working on a solution that will be available in a future version."

Long, I wonder if we should change the extension of the project files to  `.ddp3` or sthg to differentiate them on disk?  What do you think?  Kind of like microsoft went from `.doc` to `.docx` when they went to an xml serialization.  This will cause less confusion in the future and it would allow you, in principle, to associate your `.ddp` file extension with PDFguiP2 and `.ddp3` with PDFgui3.  It does mean, moving forwards, that the file extensions will be ugly, but....

Reply:
I pushed the edits on the words.

Regarding changing `.ddp` to `.ddp3`, I googled a bit, while actually I was getting again back into the rabbit hole about the pickle thing. In practice, I still feel like it is definitely do-able for python3 to read pickle files, but we just didn't figure out a good way to do it. If we succeed in this in the future (or using json or other helper function instead of pickle), it is not so necessary to change the extension name.
 
But I do agree with you that changing the extension is a good way to differentiate them. For example, Microsoft Word can open both `.doc` and `.docx`. This could be the future that Python3 PDFgui can open `.ddp` and `.ddp3`.

Reply:
I think I like the ddp3 change tbh.  If users have a directory with a bunch of ddps and ddp3's they will know exactly what to do. If we don't change the name it will be chaos and frustration....

Reply:
if later we get the p3 version to read both kinds....all the better!